### ISSUE TITLE
Regarding Skin usage in case of CSE

### ISSUE THREAD

**Issue by vinaygangaraj** [Adyen/adyen-web-sdk-sample-code#10]
Could you please let me know how we can use this skin and how to integrate in the application? Is there any sample code on using CSE for local payment methods? Please help me on this

**Comment by vinaygangaraj** [comment]
Hi,

I went through the Adyen Developers guide and i found link one is via Ecommerce and another is Checkout integration.

Under Ecommerce we have 2 options 

https://docs.adyen.com/developers/ecommerce-integration/cse-integration-ecommerce

1.	Client-Side Encryption (CSE):
	Here for Cards we have to use Encryption library and for local payment methods we need to use skin. In this scenario, can you please tell me ,
		- Local Payment Methods will also handle Cards options along with Local payments?
		- Also, How to integrate Skin in the payment form?
		- Do we need to write our own payment form or Skin itself provides the UI?
		
2.	API integration:
	For this option to use, we should support PCI compliance at Level 1 or 2.
	
	
Under Checkout also we have 2 options ,

https://docs.adyen.com/developers/checkout/integrations-overview

1.	SDKS:
This is quickest way to integrate Payment platform for Web, iOS and Android using APIs. This also provides UI.

2.	API Integration:
If we want to use our own UI or create custom payment flows, we can use the Checkout API integration. If we are a PCI SAQ-D merchant, then we can also use the API to collect the shopper's card data. 


Basically, I need to integrate Alipay(local payment method) in our application along with Cards(Credit/Debit cards) option. So, could you please let me know which method is the best for this requirement.

And, I feel Checkout SDK integration suits for my requirement. Could you please let me know on this.

Also, kinldy let me know what is the procedure to have basic PCI compliance(SAQ-A form) as the SDK integration specifies that it requires basic basic PCI compliance (SAQ-A form).

Thanks,
Vinay

**Comment by Trigalti** [comment]
Hey Vinay,

It would be best to send a request to support@adyen.com for any further questions on SAQ related topics (they can also help you choose the best way of integrating, dependent on your future plans etc.).

On the choice of integration:
CSE is meant to be used as an encryption library, and gives you full control over the UI.
If you want to use it in combination with local payment methods, you'd have to use either HPP (hosted payment pages, where skins can be uploaded to), or do a directory lookup call:

https://docs.adyen.com/developers/features/hosted-payment-pages/payment-request/directory-lookup-call

With Checkout, Alipay is supported, and the credit card form is also provided. Just note that Alipay acts as a redirect method (and you have no control over the Alipay QR-code page). 

Skinning the checkout is as simple as overriding the default skin + passing styling options to the hosted fields (these are iframes).

This is described in the following sets of documentation:
https://docs.adyen.com/developers/checkout/web-sdk/customization/styling

Kind regards, hope to have answered your questions,

Simon

**Comment by vinaygangaraj** [comment]
Thanks much for your valuable comments.
